app/src/main/java/com/andryoga/safebox/ui/common/Utils.kt

```diff
@@ -8,7 +8,7 @@ object Utils {
         views.forEach { view ->
             when (view.visibility) {
                 View.VISIBLE -> {
-                    view.visibility = View.INVISIBLE
+                    view.visibility = View.GONE
```

Comment:
This is a common method. whenever you change something in a common method always see dependents of method and check there is no regression because of this change.

Reply:
Oh I didn't pay attention towards this I will make sure it don't change anything and amend the code if required.